src/plugins/saved_objects_management/public/management_section/mount_section.tsx

```diff
@@ -167,7 +168,7 @@ const SavedObjectsTablePage = ({
   setBreadcrumbs: (crumbs: ChromeBreadcrumb[]) => void;
 }) => {
   const capabilities = coreStart.application.capabilities;
-  const itemsPerPage = coreStart.uiSettings.get<number>('savedObjects:perPage', 50);
+  const itemsPerPage = coreStart.uiSettings.get<number>(PER_PAGE_SETTING, 50);
```

Comment:
Shouldn't `saved_objects` plugin expose this as a part of the public contract? Otherwise, it creates an implicit dependency via the `ui_settings` mechanism.

Reply:
was done this way, resolving conversation